pkgs/development/tools/misc/luarocks/default.nix

```diff
@@ -43,7 +45,7 @@ stdenv.mkDerivation rec {
     fi
   '';
 
-  nativeBuildInputs = [ makeWrapper installShellFiles ];
```

Comment:
splicing isn't working correctly for `lua` :/

Reply:
probably caused by these https://github.com/NixOS/nixpkgs/blob/070314b62d7b243ee127bda8710e0020776c48a8/pkgs/development/interpreters/lua-5/interpreter.nix#L14-L16

Reply:
Can this be easily fixed?

Reply:
i'm not familiar with splicing scopes(`lib.makeScopeWithSplicing`) yet and the lua package set is different from for example steamPackages which uses `lib.makeScopeWithSplicing`

I have reverted the commit for now because it introduced a wrong lua version to the closure